week-2/extended-project/valeria_and_friends/README.md

```diff
@@ -0,0 +1,21 @@
+### Use instructions
+
+```bash
+ruby available_tops.rb
+```
+
+### Team members
+
+- Christopher
+- Paul
+- Valeria
+
+### Pending
+
+- ~~use local json in test mode~~
+- ~~hacer print~~
+- ~~generar csv~~
+- Mejorar interfaz
+- Switch using api and local storage
+- Validaciones rspec
+- Validar que son varias páginas
```

Comment:
I like your list of pending tasks. 👍

Reply:
Visual Studio Code has many plugins that identify the TODO tag. This can be very helpful to coordinate group work.